advancedTodo/src/redux/todoAppComplete/todoApp.ui.js

```diff
@@ -77,100 +78,100 @@ class TodoApp extends Component {
 	}
 
 	createNotification(type) {
-    switch (type) {
-      case 'success':
-        NotificationManager.success('Information successfully saved', 'Congratulations !');
-        break;
-      case 'deleteSuccessful':
-        NotificationManager.success('Todo Deleted Successfully', 'Congratulations !');
-        break;
-      case 'completeSuccessful':
-        NotificationManager.success('Todo Completed Successfully', 'Congratulations !');
-        break;
-      case 'error':
-        NotificationManager.error('Please add information to save', 'Warning', 5000, () => {
-          alert('Error');
-        });
-        break;
-			default :
-		   	return undefined
-    }
-  }
-
- handleSearch(e){
- 	let {newTodos} = this.props;
- 	let currentList = newTodos.filter( el => (el.name).includes(e.target.value));
- 	this.setState({
- 		currentList
- 	})
- }
-
- onFocus(e){
- 	this.setState({
- 		searchActive: true
- 	})
- }
-
- handleKeyPress(e){
- 	if(e.key === 'Enter'){
- 		this.handleOnClick()
- 	}
- }
+		switch (type) {
+			case 'success':
+				NotificationManager.success(SUCCESS_MESSAGE);
+				break;
+			case 'deleteSuccessful':
+				NotificationManager.success(DELETE_MESSAGE);

```

Comment:
It would be better to use a constant for the string literal 'orange' to improve code readability and maintainability. This would also make it easier to change the color in the future if needed.